packages/icons/src/components/Icon.tsx

```diff
@@ -31,7 +31,8 @@ export const Icon = ({ name, customSize, size = 'large', color = 'gray1000' }: I
     } = useNativeStyles();
     const sizeNumber = customSize || iconSizes[size];
     const paint = useMemo(() => Skia.Paint(), []);
-    paint.setColorFilter(Skia.ColorFilter.MakeBlend(Skia.Color(colors[color]), BlendMode.SrcIn));
+    const skiaColor = color in colors ? colors[color as Color] : color;
```

Comment:
do you need `color as Color` since you've already narrowed this type with the condition?

Reply:
Not sure why, but yes I do.
![Screenshot 2023-02-22 at 13 30 48](https://user-images.githubusercontent.com/26143964/220620335-bc366cda-493f-4be4-a841-8ec3529a36a6.png)

Reply:
Maybe if you showed the error, it would be more helpful 😅

I pushed correct usage of type predicate so you can see what the problem was. If you think it's unnecessary, revert it but there shouldn't be typecasting like this.
https://github.com/trezor/trezor-suite/pull/7685/commits/03bbc5e776f541f87b83e3f44bebd4a1579e8d51

Reply:
The whole colour definition was reworked. Not relevant anymore.